src/components/ADempiere/Field/FieldNumber.vue

```diff
@@ -180,6 +178,14 @@ export default {
     },
     customFocusGained(event) {
       this.isFocus = true
+      // this.focusGained(event)
+      this.$nextTick(() => {
+        this.$refs[this.metadata.columnName].focus()
+      })
+    },
+    select() {
+      this.$refs[this.metadata.columnName].select()
```

Comment:
Add the `nextTick` in this method, to make it focus right after rendering to avoid unexpected behavior.
```javascript
      this.$nextTick(() => {
        this.$refs[this.metadata.columnName].select()
      })
```
And delete unused comments.